iOS/classes/BleAdapter.swift

```diff
@@ -91,6 +91,21 @@ import Foundation
     func characteristicsForService(_ serviceIdentifier: Double,
                                    resolve: Resolve,
                                    reject: Reject)
+    
+    func descriptorsForDevice(_ deviceIdentifier: String,
+                              serviceUUID: String,
+                              characteristicUUID: String,
+                              resolve: Resolve,
+                              reject: Reject)
```

Comment:
W would add an empty line here.
```suggestion
                              reject: Reject)
                              
```